### PR TITLE
fix(refs: T35563) add register button to public detail view

### PR DESCRIPTION
**Pull request by muellerdemos**
https://yaits.demos-deutschland.de/T35563

This PR is the core part for this bug fix PR in regio:
https://github.com/demos-europe/demosplan-project-regio/pull/90 and simply adds the register flyout component to the entry points and adds two utility css classes which were needed for the fix.

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

File: client/js/bundles/procedure/publicIndexMapAndList.js [hunk 1]
```diff
@@ -14,9 +14,13 @@
 import { dpApi, prefixClass } from '@demos-europe/demosplan-ui'
 import DpSearchProcedureMap from '@DpJs/components/procedure/publicindex/DpSearchProcedureMap'
 import { initialize } from '@DpJs/InitVue'
+import RegisterFlyout from '@DpJs/components/user/RegisterFlyout'
 
 const stores = {}
-const components = { DpSearchProcedureMap }
+const components = {
+  DpSearchProcedureMap,
+  RegisterFlyout
```
Comment (spiess-demos) [inline client/js/bundles/procedure/publicIndexMapAndList.js:22]: I'm wondering if this is another case for registering a component globally... like we already do it with [some other components](https://github.com/demos-europe/demosplan-core/blob/21da7fe683e43c2151fadd59ef6a2d06cbeb0b4e/client/js/InitVue.js#L64-L68) that are required basically on every page.

File: demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_font-size.scss [hunk 2]
```diff
@@ -73,3 +73,7 @@
 .font-size-0 {
     font-size: 0;
 }
+
+.font-size-1_2 {
```
Comment (spiess-demos) [inline demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_font-size.scss:77]: As we are soon moving those into yaml/tailwind, i do not recommend to add it here. Also, why do we need a "one off" font size?

File: demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_height.scss [hunk 3]
```diff
@@ -48,6 +48,10 @@ $height-map-px: (
         height: 100vh; // .height-100vh
     }
 
+    &-44px {
```
Comment (spiess-demos) [inline demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_height.scss:51]: If you really need that "random" px value, try Tailwind with `h-[44px]`.

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_height.scss (hunk 3 above)
Comment (muellerdemos) [reply demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_height.scss:51]: I tried to use an [arbitrary](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values) value here but for some reason it did not work (which I couldn't find out why)

File: client/js/bundles/procedure/publicIndexMapAndList.js (hunk 1 above)
Comment (muellerdemos) [reply client/js/bundles/procedure/publicIndexMapAndList.js]: I think adding this component globally is less troublesome than adding it on multiple entry points for the same result, hence added it globally here: https://github.com/demos-europe/demosplan-core/pull/2366/commits/e0fa4600232ad27e204250d7a239759864279120

File: demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_font-size.scss (hunk 2 above)
Comment (muellerdemos) [reply demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_font-size.scss:77]: I was struggling with style overrides in the flyout component, this way appeared as simple solution to me with just minor drawbacks. However what would your proposal be how to handle this case?

Comment (spiess-demos) [reply demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_font-size.scss:77]: i see. in that case, could you move the css declaration into the project override?

File: demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_height.scss (hunk 3 above)
Comment (gruenbergerdemos) [reply demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_height.scss]: I removed the class here and used tailwind class `h-11` which fits exactly [5b6ce57](https://github.com/demos-europe/demosplan-core/pull/2366/commits/5b6ce57a2d6e31276391eb15750fbf31b1fa0f21)

File: client/js/InitVue.js [hunk 4]
```diff
@@ -40,6 +41,7 @@ Vue.use(DPVueCorePlugin)
 
 // Register components that are used globally
 Vue.component('DpObscure', DpObscure)
+Vue.component('RegisterFlyout', RegisterFlyout)
```
Comment (spiess-demos) [inline client/js/InitVue.js:44]: We could also register it in line 70.

Comment (gruenbergerdemos) [reply client/js/InitVue.js]: Done in [b700e9c](https://github.com/demos-europe/demosplan-core/pull/2366/commits/b700e9c45352bf30fcfb77b3dd5a4cd9b361e297)

**Review by spiess-demos — APPROVED** [review]
lgtm (code review)

**Pull request merged**